_faqs/2020-04-17-how-can-i-directly-open-an-image-file-in-greenshot.markdown

```diff
@@ -0,0 +1,11 @@
+---
+layout: faq
+status: publish
+published: true
+title: How can I open an image in Greenshot directly?
+tags: []
+ 
+---
+Associate an app to a file format by right clicking on an image and select *Open with* from the menu and then *Choose another app*. 
```

Comment:
```suggestion
You can easily open images in Greenshot by using the "Open image from file" and "Open image from clipboard" items from Greenshot's menu. If you also want Greenshot to be the default application for a specific image format (so that it opens e.g. when you double click an image in Windows Explorer) you can proceed as follows:
 
Associate an app to a file format by right clicking on an image and select *Open with* from the menu and then *Choose another app*. 
```